class/defaults.yml

```diff
@@ -1,4 +1,63 @@
 parameters:
   cilium:
     =_metadata: {}
-    namespace: syn-cilium
+    namespace: cilium
+    release_name: cilium
+
+    # Possible values "olm", "helm"
+    install_method: helm
+    # Possible values "opensource", "enterprise"
+    # Enterprise is currently only supported for "helm"
+    release: opensource
+
+    cilium_helm_values: &cilium_helm_values
+      cni:
+        binPath: /var/lib/cni/bin
+        confPath: /var/run/multus/cni/net.d
+      endpointRoutes:
+        enabled: true
+      hubble:
+        tls:
+          enabled: false
+      ipam:
+        mode: cluster-pool
+        operator:
+          clusterPoolIPv4MaskSize: "23"
+          clusterPoolIPv4PodCIDR: 10.128.0.0/14
+      kubeProxyReplacement: probe
+      nativeRoutingCIDR: 10.128.0.0/14
+      prometheus:
+        serviceMonitor:
+          enabled: false
```

Comment:
I assume that if a setup changes IP CIDRs in the OpenShift `install-config.yaml`, they need to be adjusted for the component as well?

Reply:
I'll add a note to that in the install documentation.

Another long-term option would be rendering the `install-config.yaml` in commodore.

Reply:
See https://github.com/appuio/openshift4-docs/pull/142/commits/e1bc1a7268a28634d24b6d0601848914941bb458

Reply:
> Another long-term option would be rendering the install-config.yaml in commodore. 

I like that idea, seems like something we should track as a story internally

Reply:
https://ticket.vshn.net/browse/OCP-305